src/Chirp.Infrastructure/CheepRepository.cs

```diff
@@ -46,6 +47,22 @@ public async Task AddCheep(int authorId, string text)
         }
     }
 
+    public void DeleteCheepsFromAuthor(int authorid){
```

Comment:
There should be "markdown" comments about the functionality in the code. 

Like so "///"